Add 15th Anniversary Art Contest newspost (#7850)
ppy/osu-wiki: news/2022-08-15-15-anniversary-art-contest.md

---
layout: post
title: "15th Anniversary Art Contest"
date: 2022-08-15 12:00:00 +0000
---

With osu!'s 15th anniversary approaching this September, it's time to take a look back at the past from the future with our latest art contest!

> ![](https://assets.ppy.sh/contests/153/header.jpg)

Reviewer: Shouldn't this be in the wikis cover folder

Author: no point uploading it twice really

> On a balmy spring evening (the 16th, to be exact) during September back in 2007, some dude by the name of Dean "peppy" Herbert released the first build of a niche rhythm game designed to emulate the gameplay of the recent cult hit *osu! Tatakae! Ouendan!* to the public.

Reviewer 2: ```suggestion
On a balmy September evening (the 16th, to be exact) back in 2007, some dude by the name of Dean "peppy" Herbert released the first build of a niche rhythm game designed to emulate the gameplay of the recent cult hit *osu! Tatakae! Ouendan!* to the public.
```

or

```suggestion
On a balmy September evening (the 16th, to be exact) back in spring 2007, some dude by the name of Dean "peppy" Herbert released the first build of a niche rhythm game designed to emulate the gameplay of the recent cult hit *osu! Tatakae! Ouendan!* to the public.
```

> The rest, is quite literally... history. A history that includes hundreds of influential creators and contributors and thousands of adoring and supportive players that have taken osu! from a pet project run by one guy on forum software that is possibly older than your parents to the sprawling and eclectic behemoth that it has become.

Reviewer 2: ignore the commas if ya feel like it

```suggestion
The rest, is quite literally... history. A history that includes hundreds of influential creators and contributors and thousands of adoring and supportive players who have taken osu! from a pet project run by one guy on forum software, that is possibly older than your parents, to the sprawling and eclectic behemoth it has become.
```

And so, as the celebration of osu!'s 15th year draws ever closer, what better way to get the festivities going than with a fan art contest of historical proportions? Thus, we present this contest's theme:

> **Draw any one or combination of the official osu! mascots (or community mascots) exploring any part of osu!'s history!**

Reviewer: > Draw any one or combination

This sounds weird to me, theres probably a smoother way to say this?

> Returning to the wide themes that made the A Place To Belong contest so interesting, we're foraying this time into something just as wide but a little more concrete. If you're a long-time player, you might already be familiar with a lot of the historical particulars, but in the event you're not, [check out the osu!wiki](/wiki/History_of_osu!) for some pointers on where to start. You may also want to consult any one of the many videos put out by content creators on the topic too.

Reviewer 2: ```suggestion
Returning to the wide themes that made the [A Place To Belong contest](https://osu.ppy.sh/community/contests/142) so interesting, we're foraying this time into something just as wide but a little more concrete. If you're a long-time player, you might already be familiar with a lot of the historical particulars, but in the event you're not, [check out the osu! wiki](/wiki/History_of_osu!) for some pointers on where to start. You may also want to consult any one of the many videos put out by content creators on the topic too.
```

Speaking of pointers, here's a few suggestions to get the juices flowing:

- The mascots enjoying iconic representations of popular maps from each game mode

> - Time-travelling mascots crying in despair after they realize they've travelled to 2016 and can no longer play any beatmaps properly

Reviewer 3: I know this is an idea, but I'm unsure what you mean by this idea and how 2016 is a major point in beatmaps.

Author: it's a joke about newer players having considerable difficulty engaging with older content in some cases (usually regarding low approach rate reading)

Reviewer 4: I believe that within the community 2016 is widely seen as the year mapping drastically changed and the likes of Monstrata, Sotarks etc., got really popular

- Mascots re-enacting the spiderman-points-at-spiderman meme with different versions of themselves wearing their old costumes (most applicable to pippi)
- Anything that celebrates some cornerstone moment in osu!'s history

> - Any one of the mascots gazing lovingly upon the faces of top players enshrined in the clouds forever-more

Reviewer 2: ```suggestion
- Any one of the mascots gazing lovingly upon the faces of top players enshrined in the clouds forevermore
```

- **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** The cold, unfeeling machine that processes your entries does not care how pretty your 690x420px submission might be, only that the numbers match.
- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** The circular spot in the middle shows where the main-menu cookie will be when inside the game. Factor this in when designing your piece, as anything inside it is likely not going to be seen. This doesn't mean leaving a big circular hole full of nothing in the middle! Be creative.

> - **Do not leave the dotted osu! cookie template visible in your submission.** Deep breaths. We're calm. It's only taken nearly twenty contests but people have finally stopped doing it.

Reviewer 2: dynamics and contrast

```suggestion
- **Do not leave the dotted osu! cookie template visible in your submission!!!** ...Deep breaths. We're calm. It's only taken nearly twenty contests but people have finally stopped doing it.
```

- **Don't use copyrighted characters from other intellectual properties.** All the official osu! mascots and the community mascots are fair game, but please avoid using representations of characters from other worlds or franchises in your work. Remember also that the community mascots are someone else's treasured creation, so please treat them with the same respect that you'd treat your own OCs and work.

---

Ready to make your mark on osu!'s history by drawing osu!'s history? Head on over to the [15th Anniversary Art Contest listing](https://osu.ppy.sh/community/contests/153) and drag-drop your entry onto the fat plus sign any time within the next **14 days**. You can also delete and resubmit your entry at will before the contest closes, so don't be shy.

> A selection of the top-voted entries will make it to the main menu as our special 15th Anniversary celebration pack, and will also score their artists **3 months of osu!supporter**, and a unique profile badge made from their own piece if they're any of the top 3.

Reviewer 2: either the Whole Thing™ or nothing

```suggestion
A selection of the top-voted entries will make it to the main menu as our special 15th anniversary celebration pack, and will also score their artists **3 months of osu!supporter**, and a unique profile badge made from their own piece if they're any of the top 3.
```
